src/main/kotlin/org/dif/DIDComm.kt

```diff
@@ -134,7 +133,15 @@ class DIDComm(private val didDocResolver: DIDDocResolver, private val secretReso
      * @return Result of pack encrypted operation.
      */
     fun packEncrypted(params: PackEncryptedParams): PackEncryptedResult {
-        return PackEncryptedResult("", listOf(), "")
+        val didDocResolver = params.didDocResolver ?: this.didDocResolver
+        val secretResolver = params.secretResolver ?: this.secretResolver
+        val senderKeySelector = SenderKeySelector(didDocResolver, secretResolver)
+
+        val (payload, signFrom) = signIfNeeded(params, senderKeySelector)
```

Comment:
Why don't make a `SignResult` data class with `payload` and `signFrom` fields?

Reply:
Because it's not sign result